openchain-blockchain-coinprism/azuredeploy.parameters.json

```diff
@@ -9,7 +9,7 @@
             "value": "Standard_A0"
         },
         "openchainVersion": {
-            "value": "0.5.0"
+            "value": "0.6.2"
```

Comment:
no harm, just unnecessary when there is a default version... any interactive deployment experience will handle the extra versions when available.